modules/next/src/NextEntityTypeManager.php

```diff
@@ -92,9 +91,11 @@ public function getEntityFromRouteMatch(RouteMatchInterface $route_match): ?Enti
    * {@inheritdoc}
    */
   public function isEntityRevisionable(EntityInterface $entity): bool {
+    // phpcs:disable
     if (\Drupal::hasService('jsonapi.resource_type.repository')) {
```

Comment:
@panshulK - Can we use DI here?

Reply:
Yes DI is a primarily way of usage for all plugins/classes/forms but I use phpcs:disable to avoid phpcs issues in our CI jobs.
Don't want to mess up the existing functionality because for D10 compatibility this is just a extra work.